mysql-test/suite/rocksdb/t/i_s_trx.test

```diff
@@ -0,0 +1,18 @@
+--source include/have_rocksdb_as_default.inc
```

Comment:
Test was renamed on merge in, this should not have been changed to _as_default.inc as the test properly uses ENGINE=ROCKSDB.